DEVELOPERS.md

```diff
@@ -0,0 +1,100 @@
+# Developer docs
+
+## Stack
+This is a [Django](https://www.djangoproject.com) project.
+It uses [Django Rest Framework](https://www.django-rest-framework.org) for the API and [SQLite](https://www.sqlite.org/index.html) for the database.
+It is deployed via [dokku](https://dokku.com), serves static files using the [whitenoise](http://whitenoise.evans.io) package, and is itself served by [gunicorn](https://gunicorn.org).
+We authenticate Users with the [Python Social Auth](https://python-social-auth.readthedocs.io) Django-specific package, using [GitHub]() as the OAuth Provider backend.
+
+Tests are run with [pytest](https://docs.pytest.org) and a selection of plug-ins.
+
+We use [black](https://black.readthedocs.io) and [isort](https://pycqa.github.io/isort/) to automatically format the codebase, with [flake8](https://flake8.pycqa.org) for linting.
+Each tool, including pytest, has been configured via config files, but a Makefile also exists to script common use cases (eg check and fix formatting).
+[pre-commit](https://pre-commit.com) is configured to run the same checks via git hooks.
+
+CI is handled by [GitHub Actions](https://github.com/features/actions) where the tests and tooling are run.
+After a successful merge to `main` a deployment is run.
+
+Errors are logged to the DataLab [Sentry](https://sentry.io) account.
```

Comment:
I'm not sure about pulling out this part of the README, it gives a technical overview but it's not necessarily useful for a developer per se, ie saying "this is a Django project" is probably not high value to someone editing the code if you're even passingly familiar with Django.  However, a technical person evaluating the platform would probably find a high level overview of the stack useful, what do you think?

Reply:
Yes, I was in two minds about this!

This section is not overwhelmingly technical. It could equally stay in the main README without putting anyone off.

Reply:
> not overwhelmingly technical

I think that's the crux of it!  Can you pull that bit back over?  Feel free to rebase it to keep history clean 👍

Reply:
Done; please take a look! :slightly_smiling_face:

Reply:
And please merge on my behalf if all's good here @ghickman!